### PR TITLE
Klaytn contract wizard

### PR DESCRIPTION
**Pull request by ayo-klaytn**
## Proposed changes
added Klaytn contracts wizard (KCW)

## Types of changes

Please put an x in the boxes related to your change.

- [x] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/toolkit/klaytn-contracts-wizard.md [hunk 1]
```diff
@@ -0,0 +1,455 @@
+# Klaytn Contracts Wizard.
```
Comment (kjhman21) [inline docs/toolkit/klaytn-contracts-wizard.md:1]:
```suggestion
# Klaytn Contracts Wizard
```

File: docs/toolkit/klaytn-contracts-wizard.md [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+* Generate and customize smart contract code using Klaytn Contracts Wizard.
+* Deploy Klaytn contracts to the Klaytn Network (Baobab) using Foundry Scripting System.
+
+## Exploring Klaytn Contracts Wizard
+
+Klaytn Contracts Wizard posits itself as the fastest and easiest way to write your smart contract using Klaytn Contracts. In this section, we will dive into the various components and segments of the Klaytn Contract Wizard.
+
+As it is, the Klaytn contracts wizard supports the following token standards:
+
+* [KIP7](https://kips.klaytn.foundation/KIPs/kip-7) — This is a fungible token standard for Klaytn. Fungible means that all tokens are divisible and interchangeable, that is, have the same value. One typical example of fungible tokens is fiat currencies, where each equal-denomination bill has the same value.
```
Comment (kjhman21) [inline docs/toolkit/klaytn-contracts-wizard.md:20]:
```suggestion
* [KIP-7](https://kips.klaytn.foundation/KIPs/kip-7) — This is a fungible token standard for Klaytn. Fungible means that all tokens are divisible and interchangeable, that is, have the same value. One typical example of fungible tokens is fiat currencies, where each equal-denomination bill has the same value.
```

we are using '-' for standard name, but we are not using '-' for the code.

File: docs/toolkit/klaytn-contracts-wizard.md [hunk 3]
```diff
@@ -0,0 +1,455 @@
...
+Let’s get started! 
+
+### Prerequisites
+
+To follow along in this tutorial, the prerequisites are highlighted below:
+Make sure to have [foundry](https://book.getfoundry.sh/getting-started/installation) installed. 
+Clone the [klaytn-foundry-starterkit](https://github.com/ayo-klaytn/klaytn-foundry-starterkit) code.
+[MetaMask](https://docs.klaytn.foundation/dapp/tutorials/connecting-metamask#install-metamask): used to deploy the contracts, sign transactions and interact with the contracts.
+RPC Endpoint: you can get this from one of the supported [endpoint providers](https://docs.klaytn.foundation/content/dapp/json-rpc/public-en).
+.Test KLAY from [Faucet](https://baobab.wallet.klaytn.foundation/faucet): fund your account with sufficient KLAY.
```
Comment (Scott-Klatn) [inline docs/toolkit/klaytn-contracts-wizard.md:59]:
```suggestion
* Make sure to have [foundry](https://book.getfoundry.sh/getting-started/installation) installed. 
* Clone the [klaytn-foundry-starterkit](https://github.com/ayo-klaytn/klaytn-foundry-starterkit) code.
* [MetaMask](https://docs.klaytn.foundation/dapp/tutorials/connecting-metamask#install-metamask): used to deploy the contracts, sign transactions and interact with the contracts.
* RPC Endpoint: you can get this from one of the supported [endpoint providers](https://docs.klaytn.foundation/content/dapp/json-rpc/public-en).
* Test KLAY from [Faucet](https://baobab.wallet.klaytn.foundation/faucet): fund your account with sufficient KLAY.
```

File: docs/toolkit/klaytn-contracts-wizard.md [hunk 4]
```diff
@@ -0,0 +1,455 @@
...
+
+### Customizing Token contracts 
+
+**Customizing KIP7 contract to KIP7 Airdrop contract.** 
+You need to customize your KIP7 contract before modifying it to an airdrop contract. To do that, follow the steps below: 
+
+* Navigate to [wizard.klaytn.foundation](https://wizard.klaytn.foundation/).
+* On the **Contracts** tab select **KIP7**
+* Next is to fill the name (KIP7 Token Airdrop) and symbol (KTA) in the **SETTINGS** tab. The pre-mint field is left empty
+* Subsequently on the **FEATURES** tab, tick the **Mintable** feature box, it then automatically selects the Ownable feature in **ACCESS CONTROL** tab.
```
Comment (Scott-Klatn) [inline docs/toolkit/klaytn-contracts-wizard.md:72]: An ordered list preferred for the steps to follow.
```suggestion
1. Navigate to [wizard.klaytn.foundation](https://wizard.klaytn.foundation/).
2. On the **Contracts** tab select **KIP7**
3. Next is to fill the name (KIP7 Token Airdrop) and symbol (KTA) in the **SETTINGS** tab. The pre-mint field is left empty
4. Subsequently on the **FEATURES** tab, tick the **Mintable** feature box, it then automatically selects the Ownable feature in **ACCESS CONTROL** tab. 
```

File: docs/toolkit/klaytn-contracts-wizard.md [hunk 5]
```diff
@@ -0,0 +1,455 @@
...
+Now that we have our KIP7 airdrop contract code ready, the next step is to create a new file named airdropKIP7.sol in the src folder of your project directory and paste the modified code in the file.
+
+**Customizing KIP17 contract to KIP17 Airdrop contract.**
+
+You need to customize your KIP17 contract before modifying it to an airdrop contract. To do that, follow the steps below: 
+
+* Navigate to [wizard.klaytn.foundation](https://wizard.klaytn.foundation/). 
+* On the **Contracts** tab select **KIP17**
+* Next is to fill the name (KIP7 NFT Airdrop) and symbol (KNA) in the **SETTINGS** tab.  The Base URI field is to be left empty. 
+* Subsequently on the **FEATURES** tab, tick the **Mintable**, **Auto-increment Ids**, and **Enumerable** feature box. You will notice that the Ownable feature in **ACCESS CONTROL** tab has been automatically selected.
```
Comment (Scott-Klatn) [inline docs/toolkit/klaytn-contracts-wizard.md:149]:
```suggestion
1. Navigate to [wizard.klaytn.foundation](https://wizard.klaytn.foundation/). 
2. On the **Contracts** tab select **KIP17**
3. Next is to fill the name (KIP7 NFT Airdrop) and symbol (KNA) in the **SETTINGS** tab.  The Base URI field is to be left empty. 
4. Subsequently on the **FEATURES** tab, tick the **Mintable**, **Auto-increment Ids**, and **Enumerable** feature box. You will notice that the Ownable feature in **ACCESS CONTROL** tab has been automatically selected. 
```

File: docs/toolkit/klaytn-contracts-wizard.md [hunk 6]
```diff
@@ -0,0 +1,455 @@
...
+Now that we have our KIP17 airdrop contract code ready, the next step is to create a new file named airdropKIP17.sol in the src folder of your project directory and paste the modified code in the file.
+
+**Customizing KIP37 contract.**
+
+Because KIP37 supports batch minting, we will only customize the contract and use it as it is. To customize our KIP37Contract, follow the steps below: 
+
+* Navigate to [wizard.klaytn.foundation.](https://wizard.klaytn.foundation/) 
+On the **Contracts** tab select **KIP37**
+Next is to fill the name (KIP7 NFT Airdrop) and symbol (KNA) in the **SETTINGS** tab.  The Base URI field is to be left empty. 
+Subsequently on the **FEATURES** tab, tick the **Mintable**, **Auto-increment Ids**, and **Enumerable** feature box. You will notice that the Ownable feature in **ACCESS CONTROL** tab has been automatically selected.
```
Comment (Scott-Klatn) [inline docs/toolkit/klaytn-contracts-wizard.md:247]:
```suggestion
1. Navigate to [wizard.klaytn.foundation.](https://wizard.klaytn.foundation/) 
2. On the **Contracts** tab select **KIP37**
3. Next is to fill the name (KIP7 NFT Airdrop) and symbol (KNA) in the **SETTINGS** tab.  The Base URI field is to be left empty. 
4. Subsequently on the **FEATURES** tab, tick the **Mintable**, **Auto-increment Ids**, and **Enumerable** feature box. You will notice that the Ownable feature in **ACCESS CONTROL** tab has been automatically selected. 
```

File: docs/toolkit/klaytn-contracts-wizard.md [hunk 7]
```diff
@@ -0,0 +1,455 @@
...
+        uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+
+        vm.startBroadcast(deployerPrivateKey);
+
+        KIP37MultiToken kip37MultiToken = new KIP37MultiToken();
+
+        vm.stopBroadcast();
+    }
+}
+
```
Comment (Scott-Klatn) [inline docs/toolkit/klaytn-contracts-wizard.md:394]: Closing backticks for the code block omitted.
```suggestion
```
```

**Review by Scott-Klatn — APPROVED** [review]

**Review by praveen-klaytn — APPROVED** [review]

**Review by kjhman21 — APPROVED** [review]

**Pull request merged**